wepool-app/src/screens/StartScreens/StartScreen.tsx

```diff
@@ -42,7 +42,8 @@ export const StartScreen = ({
     <View style={styles.container}>
       <Logo />
       <Header text="Welcome!"></Header>
-      <TouchableOpacity style={styles.buttonGPlusStyle} activeOpacity={0.5} onPress={() => promptAsync()}>
+      {/* <TouchableOpacity style={styles.buttonGPlusStyle} activeOpacity={0.5} onPress={() => promptAsync()}> */}
```

Comment:
I know login is not working properly in some platforms, but let's not remove it altogether from the main branch.